pkg/controllers/data.go

```diff
@@ -45,38 +48,44 @@ func (d *DataController) GetAll(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	things, err := d.DataInteractor.GetAll(order, skip, take, startDate, finishDate)
+	token := r.Header.Get("Authorization")
```

Comment:
In order to keep compatible with the [previous API](https://github.com/CESARBR/knot-cloud-storage#header-parameters-1), I think the header could be `auth_token`. What do you think?

Reply:
Agreed! Done :smile: